src/routing/wrappers/Layout.svelte

```diff
@@ -29,9 +32,9 @@
       <MainSelectors />
     </ContentHeader>
     <ContentBody>
-      <div class="graphic-body">
+      <div class="graphic-body" class:graphic-body-home={isHomePageView}>
         <slot />
-        {#if $store.route.section === 'probe'}
+        {#if !isHomePageView}
```

Comment:
What's the benefit of reversing this logic? We're rendering something that only makes sense in the probe section.